src/main/groovy/io/seqera/wave/service/blob/impl/BlobCacheServiceImpl.groovy

```diff
@@ -82,6 +83,9 @@ class BlobCacheServiceImpl implements BlobCacheService {
     @Inject
     private HttpClientConfig httpConfig
 
+    @Inject
+    private CleanupStrategy cleanup
```

Comment:
I think this should not be exposed as a service method and only kept in the K8s implementtion. Thoughts?

Reply:
sorry, I dont understand
CleanupStrategy only provides information about whether cleanup is set for onsuccess, true or false. 
you want to remove CleanupStrategy?

Reply:
My point is that the job can be deleted within thr

Reply:
ok noted

Reply:
done